packages/frontend/src/components/common/Table/Table.styles.ts

```diff
@@ -106,8 +107,15 @@ export const TableFooter = styled.div`
     margin-top: 10px;
 `;
 
+const FontSyles = `
+    font-size: 13px;
+    font-family: Inter, sans-serif;
+`;
+
 const CellStyles = css<{ $isNaN: boolean }>`
     text-align: ${({ $isNaN }) => ($isNaN ? 'left' : 'right')} !important;
+    padding: 0.5rem !important;
```

Comment:
I agree in general, but the goal here was to make it match the pivot table which uses rems. Should we change both?

Reply:
personally, I think it's better to get rid of them 😅

Reply:
The pivot table style is coming from Mantine, so I can't really change it in a straightforward way. I could change this one to be *pretty similar* to the pivot table by putting something like 7px. It will be a fractional pixel count and it won't be perfect, but if we are avoiding rem in our code, that's likely the best option.

Reply:
Done

Reply:
mantine 5 uses rems but since this is a mix of styled components + blueprint table let's just go with PX. I'm kinda against hardcoding rem in css